packages/protocol/contracts/rewards/IProtocolRewards.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+interface IProtocolRewards {
+
+    error ADDRESS_ZERO();
```

Comment:
error format we're following is `ContractName_ErrorName` , it should be something like `ProtocolRewards_AddressZero`

Reply:
+1